### PR TITLE
Add bastion allow list configuration

### LINKED ISSUE #71: User has to rerun step 5 to add their user to bastion access list
## Expected Behavior

when the user is in step 6 they are able to access the bastion host

## Actual Behavior

Process for adding the user to the bastion access list is not clear in the execution of step 5 and the user has either to do the manual configuration or to find where to update the configuration in step 5 and rerun step 5.

## Steps to Reproduce the Problem

1. Deploy up to step 5
1. Start the deploy of step 6

## Specifications

- Version: de00f4fe2540912949a6fc32691c14964211c2d3
- Platform: 20.04

### PR DESCRIPTION
**Pull request by daniel-cit**
Fixes #71 

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: 5-infrastructure/business_unit_1/non-production/variables.tf [hunk 1]
```diff
@@ -96,6 +96,12 @@ variable "bin_auth_attestor_project_id" {
   description = "Project id where binary attestors are created (app_cicd project from shared)"
 }
 
+variable "bastion_members" {
+  type        = list(string)
+  description = "The names of the members of the bastion server."
```
Comment (bharathkkb) [inline 5-infrastructure/business_unit_1/non-production/variables.tf:101]:
```suggestion
  description = "The emails of the members with access the bastion server."
```

File: 6-anthos-install/README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Deploying Bank of Anthos
-These instructions need to be run on the bastion host. To access the bastion host, open the Google Cloud console, and navigate to Compute Engine in the GKE project `prj-bu1-d-boa-gke`. Then, select the `gce-bastion-us-west1-b-01` and click on `ssh`. You need to be a whitelisted member to access this node which can be achieved either [manually](https://cloud.google.com/iap/docs/using-tcp-forwarding#grant-permission) or by adding your GCP User Identity in the list of `bastion_members` in `5-infrastructure/business_unit_1/<environment>/main.tf`
+
+These instructions need to be run on the bastion host.
```
Comment (bharathkkb) [inline 6-anthos-install/README.md:3]: Can we make this a h3/h4 so it wont be missed with other txt?

**Review by bharathkkb — APPROVED** [review]

File: 5-infrastructure/business_unit_1/non-production/variables.tf (hunk 1 above)
Comment (daniel-cit) [reply 5-infrastructure/business_unit_1/non-production/variables.tf]: changed in all related files

File: 6-anthos-install/README.md (hunk 2 above)
Comment (daniel-cit) [reply 6-anthos-install/README.md]: Layout changed to highlight the instructions

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**